src/core/QCheck2.ml

```diff
@@ -399,14 +412,16 @@ module Gen = struct
       let right = RS.bits st in
       left lor middle lor right
 
-  let pint ?(origin : int = 0) : int t = fun st ->
+  let nat_origin origin : int t = fun st ->
     let x = pint_raw st in
```

Comment:
I suggest renaming `pint_raw` to `nat_pos_raw` while we are at it.
I know it isn't visible from the outside, however using the naming principles internally should make reading the implementation easier for ourselves and others going forward.